app/api/monthly.py

```diff
@@ -0,0 +1,448 @@
+from app import token
+from app import mongo
+from app.util import serialize_doc, get_manager_profile
+from flask import (
+    Blueprint, flash, jsonify, abort, request
+)
+import requests
+from app.config import attn_url, secret_key
+import json
+import dateutil.parser
+from bson.objectid import ObjectId
+
+from app.util import slack_message, slack_msg
+from app.config import slack_token
+from slackclient import SlackClient
+
+import datetime
+from flask_jwt_extended import (
+    JWTManager, jwt_required, create_access_token,
+    get_jwt_identity, get_current_user, jwt_refresh_token_required,
+    verify_jwt_in_request
+)
+from app.config import slack_token
+
+bp = Blueprint('monthly', __name__, url_prefix='/')
+
+# function for getting all the juniors of managers
```

Comment:
this function need to be common. we cannot write this function everytime we need it. it used in 360 also. make it common in app.util

Reply:
sir in 360 he is finding managers of a particular user here i am finding juniors of a particular user they both are diffrent